Analysis/PIDTools/src/LikelihoodPIDProcessor.cc

```diff
@@ -423,7 +422,8 @@ void LikelihoodPIDProcessor::createParticleIDClass(int parttype, ReconstructedPa
   Double_t *posterior = _myPID->GetPosterior();
   Double_t *likelihood = _myPID->GetLikelihood();
   std::vector<float> likelihoodProb;
-  if(pidh.getAlgorithmName(algoID)!="LowMomMuID"){
+
+  if(pidh.getAlgorithmName(algoID).find("LowMomMuID") == std::string::npos){
```

Comment:
Here is the second part where I am not sure if it is a typo or is as it should be.